### ISSUE TITLE
[BadDomList] Broken Tests (Trac #1705)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1697]
The tests use data from the Data Warehouse.  That needs to be included in I3_TESTDATA.

Migrated from https://code.icecube.wisc.edu/ticket/1705
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:12:58", 
    "description": "The tests use data from the Data Warehouse.  That needs to be included in I3_TESTDATA.", 
    "reporter": "olivas", 
    "cc": "nega", 
    "resolution": "fixed", 
    "_ts": "1550067178841456", 
    "component": "infrastructure", 
    "summary": "[BadDomList] Broken Tests", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2016-05-16T21:37:30", 
    "milestone": "", 
    "owner": "joertlin", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160517T20:13:16: 
* joertlin commented:

> In changeset 145995:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="145995"
> Changed input file path from data warehouse to $I3_TESTDATA for tests. Closes #1697
> ```


* joertlin changed resolution from "" to "fixed"
* joertlin changed status from "new" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:12:58: 
* olivas changed milestone from "May 2016" to ""
* olivas commented:

> Milestone May 2016 deleted
